airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml

```diff
@@ -0,0 +1,876 @@
+            parent_stream_configs:
+              - type: ParentStreamConfig
+                parent_key: id
+                partition_field: project_id
+                stream:
+                  $ref: "#/definitions/streams/projects"
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/labels"
+    releases:
+      type: DeclarativeStream
+      name: releases
+      primary_key:
+        - id
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: projects/{{ stream_slice['project_id'] }}/releases
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+        partition_router:
+          - type: SubstreamPartitionRouter
+            parent_stream_configs:
+              - type: ParentStreamConfig
+                parent_key: id
+                partition_field: project_id
+                stream:
+                  $ref: "#/definitions/streams/projects"
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/releases"
+    epics:
+      type: DeclarativeStream
+      name: epics
+      primary_key:
+        - id
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: projects/{{ stream_slice['project_id'] }}/epics
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+        partition_router:
+          - type: SubstreamPartitionRouter
+            parent_stream_configs:
+              - type: ParentStreamConfig
+                parent_key: id
+                partition_field: project_id
+                stream:
+                  $ref: "#/definitions/streams/projects"
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/epics"
+    activity:
+      type: DeclarativeStream
+      name: activity
+      primary_key:
+        - guid
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: projects/{{ stream_slice['project_id'] }}/activity
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestOption
+            inject_into: request_parameter
+            field_name: offset
+          page_size_option:
+            type: RequestOption
+            field_name: limit
+            inject_into: request_parameter
+          pagination_strategy:
+            type: OffsetIncrement
+            page_size: 10
```

Comment:
I think this parameter should be taken from the response - please see [doc](https://www.pivotaltracker.com/help/api#Paginating_List_Responses). Also, this param should not be included on the first call

Reply:
the page_size is integer type variable and cannot be changed with jinja macros in Yaml
Even if we try cursor pagination, for getting the value, the page_size is strictly typed
![image](https://github.com/airbytehq/airbyte/assets/47392334/3e8ef7cb-4a9e-4790-871d-4542f08471df)

Reply:
Is there any neccesity for changing this? Current pagination strategy works perfectly
Ref: Builder - Partition 2
![image](https://github.com/airbytehq/airbyte/assets/47392334/51dc3e60-e883-41fe-ae8f-138819430804)

Builder - Partition 1
![image](https://github.com/airbytehq/airbyte/assets/47392334/d75df227-48ba-43ca-9cfd-00fd720ac9cb)

Reply:
Docs says ``
![image](https://github.com/airbytehq/airbyte/assets/47392334/14b022cc-e13b-4522-b13f-a93d92c10518)
`If no limit parameter is supplied for a paginated request, a default limit value will be used`